src/cli/test_command.zig

```diff
@@ -572,11 +615,21 @@ pub const TestCommand = struct {
     pub const name = "test";
     pub const CodeCoverageOptions = struct {
         skip_test_files: bool = !Environment.allow_assert,
+        reporters: []const Reporter = &.{ Reporter.console },
```

Comment:
I think instead of a slice, we should just have a struct of all of the coverage reporters.

```
const Reporters = struct {
  console: bool,
  lcov: bool,
};
```

The current way you have makes it possible to specify a coverage reporter more than once, which duplicates work and is likely unintentional. This also would reduce allocations.

Reply:
Did so in https://github.com/oven-sh/bun/pull/11883/commits/9ab1a772844f6e02f285a68b01f783556c974d7c